piqueserver/config/config.toml

```diff
@@ -168,6 +168,12 @@ time_announcements = [ 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 30, 60, 120, 180, 240, 300
 # number of intel captures to win the game (default: 10)
 cap_limit = 10
 
+# Enable command antispam for preventing users to abuse from flooding commands
+enable_command_antispam = true
+
+# Maximum commands to be executed in the given time
+max_command_size = 4
```

Comment:
Without reading the comment supplied with the setting, it may seem like 'max command size = how long a command can be; only 4 characters'. Perhaps adding a plural (s character) would help: `command_ratelimit_amount`.